libs/utils/android/system.py

```diff
@@ -21,9 +21,14 @@
 from devlib import TargetError
 import os
 import pexpect as pe
+from time import sleep
 
 GET_FRAMESTATS_CMD = 'shell dumpsys gfxinfo {} > {}'
 
+# See https://developer.android.com/reference/android/content/Intent.html#setFlags(int)
+FLAG_ACTIVITY_NEW_TASK = 268435456
```

Comment:
I didn't know how python would output hexes so I went with dead stupid integers, I'll fix that